sites/geohub/package.json

```diff
@@ -123,19 +125,22 @@
 		"svelte-use-click-outside": "^1.0.0",
 		"swagger-ui-dist": "^5.10.5",
 		"tippy.js": "^6.3.7",
+		"topojson-client": "^3.1.0",
 		"tslib": "^2.6.2",
 		"typescript": "^5.3.3",
 		"uuid": "^10.0.0",
 		"valid-filename": "^4.0.0",
 		"vite": "^5.0.0",
-		"vitest": "^2.0.0"
+		"vitest": "^2.0.0",
+		"xlsx": "^0.18.5"
 	},
 	"dependencies": {
 		"@azure/service-bus": "^7.9.3",
 		"@azure/storage-blob": "^12.17.0",
 		"@azure/web-pubsub": "^1.1.1",
 		"@mapbox/geo-viewport": "^0.5.0",
 		"@mapbox/vector-tile": "^2.0.0",
+		"@turf/turf": "^7.0.0",
```

Comment:
@lrt-sibat Please move `@turf/turf` to `devDependencies`. libraries used in frontend should not be included in `dependencies`.

Also, please consider using [@turf/bbox](https://www.npmjs.com/package/@turf/bbox) other adding the heavy turf.js package if you only need `bbox` feature.